zproject/prod_settings_template.py

```diff
@@ -453,6 +453,57 @@
 # Similarly if you want to set a Privacy Policy.
 #PRIVACY_POLICY = '/etc/zulip/privacy.md'
```

Comment:
I think we want to have all the service configurations next to each other; this move has TERMS_OF_SERVICE and PRIVACY_POLICY sandwiched between such settings.

Reply:
Moved TERMS_OF_SERVICE and PRIVACY_POLICY at the end of the Miscellaneous settings.